### ISSUE TITLE
`bbolt` CLI util panics / shows not-user-friendly errors in certain cases with incorrect (number of) arguments 

### ISSUE THREAD

**Issue by audibleptr** [etcd-io/bbolt#681]
I see there's lots of issues about panics, but I think this one is different, as it relates to the CLI util, rather than the library.

For at least several commands, the CLI utility doesn't properly handle missing arguments, resorting to various panics and errors, rather than printing useful error messages to the user.

For example:

```
$ bbolt get
panic: runtime error: slice bounds out of range [:-1]

goroutine 1 [running]:
main.(*getCommand).Run(0xc00006cdc8, {0xc000014070, 0x0, 0x0})
        /usr/src/RPM/BUILD/bbolt-1.3.8/cmd/bbolt/main.go:996 +0x699
main.(*Main).Run(0xc00006cf40, {0xc000014070?, 0x7f66b3c44108?, 0x60?})
        /usr/src/RPM/BUILD/bbolt-1.3.8/cmd/bbolt/main.go:123 +0x10d
main.main()
        /usr/src/RPM/BUILD/bbolt-1.3.8/cmd/bbolt/main.go:67 +0xae
```

```
$ bbolt page-item test.db 4
strconv.ParseUint: parsing "": invalid syntax
```

The DB itself is valid, and with correct arguments `bbolt` returns correct results.

`bbolt 1.3.8-alt1` on ALT Linux

**Comment by ahrtr** [comment]
Thanks for raising this issue.

@Elbehery could you fix this issue, and backport it to release-1.3?

**Comment by Elbehery** [comment]
sure, on it 👍🏽

**Comment by Elbehery** [comment]
raised backport-1.3 at https://github.com/etcd-io/bbolt/pull/683

**Comment by audibleptr** [comment]
Hello, thanks for the fix, however, the second case I mentioned does not seem to be fixed as of `bbolt` `1.4.2`:

```console
$ etcd & etcdctl put foo bar; etcdctl put abc def; pkill etcd
$ bbolt page-item default.etcd 4
strconv.ParseUint: parsing "": invalid syntax
```

**Comment by ahrtr** [comment]
> Hello, thanks for the fix, however, the second case I mentioned does not seem to be fixed as of `bbolt` `1.4.2`:
> 
> $ etcd & etcdctl put foo bar; etcdctl put abc def; pkill etcd
> $ bbolt page-item default.etcd 4
> strconv.ParseUint: parsing "": invalid syntax

I believe it's been fixed on main branch. The fix will be included in v1.5.0, which hasn't been released yet.

```
$ ./bbolt page-item ~/tmp/etcd/infra2.etcd/member/snap/db 4  
Error: accepts 3 arg(s), received 2
Usage:
  bbolt page-item [options] <bbolt-file> pageid itemid [flags]

Flags:
      --format string   Output format one of: auto|ascii-encoded|hex|bytes|redacted (default "auto")
  -h, --help            help for page-item
      --key-only        Print only the key
      --value-only      Print only the value
```
